# Values 0..7
uint8[] valve
uint16[] target
